### PR TITLE
Fix topBar background color on iOS 13

### PR DESCRIPTION
**Pull request by yogevbd**

### CODE REVIEW

File: lib/ios/UINavigationBar+utils.h [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#import <UIKit/UIKit.h>
+
+@interface UINavigationBar (utils)
+
+- (void)rnn_setBackgroundColor:(UIColor *)color;
```
Comment (guyca) [inline lib/ios/UINavigationBar+utils.h]: Why do we append the `rnn_` prefix to these methods? Can we avoid prefixing methods?

File: lib/ios/RNNStackController.m [hunk 2]
```diff
@@ -52,35 +51,7 @@ - (UIViewController *)childViewControllerForStatusBarStyle {
...
-		}
-	} else {
-		UIView *transparentView = [self.navigationBar viewWithTag:TOP_BAR_TRANSPARENT_TAG];
-		if (transparentView){
-			[transparentView removeFromSuperview];
-		}
-		
-		self.navigationBar.barTintColor = nil;
-	}
+	[self.navigationBar rnn_setBackgroundColor:backgroundColor];
```
Comment (guyca) [inline lib/ios/RNNStackController.m]: WOW! I love that you move this entire function to a category!

**Comment by jfrolich** [comment]
Why is this dropping support for a transparent background color?

**Comment by guyca** [comment]
@jfrolich W00ps, that's unintentional. We'll fix asap

**Pull request merged**